cyences_app_for_splunk/default/data/ui/views/cs_windows_patch.xml

```diff
@@ -100,14 +94,8 @@
         <search>
           <query>`cs_wineventlog_system` host="$host$" $tkn_host_filter$ $tkn_patch_numbers$ EventCode=19 ("Microsoft Endpoint Protection" OR "Microsoft Defender Antivirus") 
 | eval message=replace(Message, "^Installation Successful: Windows successfully installed the following update: ", "") 
-| append 
-    [| inputlookup all_windows_hosts.csv 
-    | search host="$host$" $tkn_host_filter$ 
-    | eval no_event="No Update Event Found"] 
```

Comment:
This part will add additional info for the host which does not have any update event.